tests/PHPUnit/System/expected/test_CustomEvents_Events.getCategory_lastN__API.getProcessedReport_day.xml

```diff
@@ -181,7 +181,7 @@
 		<nb_events>69</nb_events>
 		<nb_events_with_value>14</nb_events_with_value>
 		<sum_event_value>137.96</sum_event_value>
-		<min_event_value>9.66</min_event_value>
+		<min_event_value>28.32</min_event_value>
```

Comment:
There must be something wrong here:

`nb_events_with_value`: 14
`min_event_value`: 28.32
`sum_event_value`: 137.96

Check: 
`nb_events_with_value` * `min_event_value` <= `sum_event_value`

- Fails: 14 * 28.32 = 396.48 is bigger than 137.96